rmc-docs/src/SUMMARY.md

```diff
@@ -6,8 +6,11 @@
   - [RMC on a single file]()
   - [RMC on a crate]()
   - [Debugging failures]()
-  
+  - [Debugging non-termination]()
```

Comment:
Is this section coming in the next batch? If so, should we take it out from this PR?

Reply:
mdbook supports "outlining" like this as a specifically added feature. Seems nice to use, and it makes the planned structure visible.